docs/docs/migrate/keptnapp/index.md

```diff
@@ -0,0 +1,60 @@
+# Migrating KeptnApp to KeptnAppContext
+
+Keptn has recently orphaned the concept of manually created KeptnApp
+and instead uses the KeptnApp auto-discovery feature combined with
+the newly introduced KeptnAppContext custom resource.
+
+## Migration steps
+
+> **Warning**
+Please note that the migration is needed to be executed only for
+**manually created KeptnApp** resources.
+KeptnApp resources created by
+the automatic app-discovery feature do not need any actions.
+
+To migrate an existing **manually created KeptnApp** to KeptnAppContext
+you need to execute the following steps:
+
+1. Create a KeptnAppContext custom resource with the same name
+as your KeptnApp has
+2. Move pre/post-deployment tasks and evaluation from KeptnApp
+to KeptnAppContext
+3. Introduce a `app.kubernetes.io/managed-by: keptn` annotation
+to KeptnApp
+4. Change the API version of KeptnApp to `lifecycle.keptn.sh/v1beta1`
+
+> **Note**
+Please make sure all of your application resources
+(such as Pods, Deployments, StatefulSets or DaemonSets)
+have the proper annotations/labels set.
```

Comment:
So do you men that all resources listed in the `KeptnApp` `spec.workoads` require these annotations?  I think it would be clearer to reference them this way rather than as "application resources", which is not the terminology that is used elsewhere.

Reply:
not particulary the resources listed in spex.workloads, but the resources, which are represented by the workloads (Deployments, StatefulSets, Pods...)

Reply:
Okay, when we get everything updated, we can look to be sure that our terminology is consistent.  It may be that some of the existing docmentation needs to switch terminology.  I am resolving the comment.